### ISSUE TITLE
More options menu: Absence of vocalization

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#2026]
## Bug Report

**Problematic behavior**
When using the "more options" menu, the actions for pinning and duplicating a document or copying as markdown using the keyboard are not announced and there is no visual confirmation that the action has been successfully completed.

On Mac, toasts are not vocalized and are not visible.
On Windows, we have "docs alert" before each alert that is vocalized.

For duplication, there is only "docs alert" with no details about the action performed.

**Expected behavior/code**
When a document is pinned or duplicated, it must be announced and/or have a toast message indicating that the action has been successfully completed.

**Additional context/Screenshots**

<img width="318" height="315" alt="Image" src="https://github.com/user-attachments/assets/84222343-45a3-45d7-b3d0-479ec4c2c0de" />

**Critères RGAA**

**Critère 7.1 :** [[Chaque script est-il, si nécessaire, compatible avec les technologies d'assistance ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#7.1)](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#7.1)

**Impact** 

A blind or visually impaired user who activates the "pin", "duplicate" or "copy as markdown" button using the keyboard has no feedback confirming that the action has been successfully completed. They must search through the page for confirmation and even then it's not always obvious.
